content/departments/engineering/process/sdlc.md

```diff
@@ -1,35 +1,78 @@
 # Software development life cycle
 
-<span class="badge badge-note">SOC2/GN-98</span>
+<span class="badge badge-note">SOC2/CI-98</span>
```

Comment:
I wonder if we should keep both badges for now, for ease of reference?

Reply:
I was thinking about updating all of them as that's the new refs we're using internally. Why would we keep the old ones?